include/mbedtls/config.h

```diff
@@ -2019,6 +2019,15 @@
  */
 //#define MBEDTLS_USE_PSA_CRYPTO
 
+/**
+ * \def MBEDTLS_PSA_CRYPTO_CONFIG
+ *
+ * This setting should be used to allow for conditional inclusion of PSA features.
+ *
```

Comment:
The description should say that the feature is still “experimental”, and explain that it's not ready for production yet because it's still incomplete.